dev/devicelab/lib/tasks/gallery.dart

```diff
@@ -10,6 +10,16 @@ import '../framework/devices.dart';
 import '../framework/framework.dart';
 import '../framework/task_result.dart';
 import '../framework/utils.dart';
+import 'build_test_task.dart';
+
+final Directory galleryDirectory = dir('${flutterDirectory.path}/dev/integration_tests/flutter_gallery');
+
+/// Temp function during gallery tests transition to build+test model.
+///
+/// https://github.com/flutter/flutter/issues/103542
+TaskFunction createGalleryTransitionBuildTest(List<String> args, {bool semanticsEnabled = false}) {
```

Comment:
LGTM. Can we add enable one of these tests in presubmit to ensure this won't break the tree?

Reply:
These tests are not configured by .ci.yaml, but the devicelab_staging_config.star. So will not affect the tree.